src/detect-lua.c

```diff
@@ -89,12 +91,18 @@ void DetectLuaRegister(void)
 #ifdef UNITTESTS
     sigmatch_table[DETECT_LUA].RegisterTests = DetectLuaRegisterTests;
 #endif
-    g_smtp_generic_list_id = DetectBufferTypeRegister("smtp_generic");
 
-    DetectAppLayerInspectEngineRegister("smtp_generic", ALPROTO_SMTP, SIG_FLAG_TOSERVER, 0,
-            DetectEngineInspectGenericList, NULL);
-    DetectAppLayerInspectEngineRegister("smtp_generic", ALPROTO_SMTP, SIG_FLAG_TOCLIENT, 0,
-            DetectEngineInspectGenericList, NULL);
+    g_lua_ja3_list_id = DetectBufferTypeRegister("lua.ja3");
+    DetectAppLayerInspectEngineRegister("lua.ja3", ALPROTO_TLS, SIG_FLAG_TOSERVER,
```

Comment:
I need a list id with both QUIC and TLS, and without prefilter engine, right ?

Reply:
indeed. Nit: I would call it `ja3.lua` as we already have `ja3.hash` etc

Reply:
ok for the naming